proposed/0006-Launcher.md

```diff
@@ -0,0 +1,204 @@
+- Feature Name: SAFE-Launcher
+- Type: New Product
+- Related components: [safe_client](https://github.com/maidsafe/safe_client), [safe_nfs](https://github.com/maidsafe/safe_nfs)
+- Start Date: 11-September-2015
+
+# Summary
+
+Launcher will be a gateway for any App that wants to work on the SAFE-Network on a user's behalf.
+
+# Motivation
+
+## Why?
+
+App's access of the SAFE-Network on behalf of the user is an issue with high security concerns. Further, decentralisation allows for uniformity in experience on various machines concerning user installed Apps.
+
+## What cases does it support?
+
+SAFE-Launcher
+
+<1> will allow user to create an account and/or log into the SAFE-Network.
+
+<2> will authenticate a user installed App to access SAFE-Network on the user's behalf.
+
+<3> will manage metadata related to Apps to give uniformity in experience when shifting from one machine to another - eg., if App `A` is installed in machine 1 then when the user logs into machine 2 using his/her SAFE-Account via Launcher, he/she will be presented with a union of all the Apps that were installed on all the machines which access the SAFE-Network on his/her behalf.
+
+<4> Not allow Apps to mutate each other's configs.
+
+<5> Easily Revoke App's ability to Read and Mutate the Network on User's behalf.
+
+## Expected outcome
```

Comment:
This whole section is a duplicate of the one above.  I'd suggest either making it meaningfully different, or removing it.